#hw01-gradle первое дз

AMANDZHI opened the pull request:
- добавлен gitignore
- добавлен gradle wrapper
- создал модуль hw01-gradle с классом HelloOtus
- сконфигурированы settings.gradle и build.gradle файлы. Подключена guava

AMANDZHI committed:
#hw01-gradle первое дз

- добавлен gitignore
- добавлен gradle wrapper
- создал модуль hw01-gradle с классом HelloOtus
- сконфигурированы settings.gradle и build.gradle файлы. Подключена guava

stvort commented on build.gradle:
```diff
@@ -0,0 +1,36 @@
+            imports {
+                mavenBom("org.springframework.boot:spring-boot-dependencies:2.5.2")
+            }
+            dependency("com.google.guava:guava:28.2-jre")
```
1. По заданию было нужно подключить последнюю версию библиотеки
2. Лучше вынести версии в отдельный интерфейс, как предлагалось в примере к занятию)

AMANDZHI replied:
1. Подключил свежую версию
2. Вынес

AMANDZHI committed:
#hw01-gradle правки замечаний по результатам ревью

- добавлен интерфейс с версиями
- использованы интерфейс с версиями в build.gradle
